sdk/v1/client.go

```diff
@@ -229,6 +229,34 @@ func (cli *Client) GetJob(ctx context.Context, reqheader client.Headers, jobID s
 	return &respHeaders, &job, nil
 }
 
+func (cli *Client) GetJobs(ctx context.Context, reqheader client.Headers) (*client.RespHeaders, *models.Jobs, error) {
+	if reqheader.ServiceAuthToken == "" {
+		reqheader.ServiceAuthToken = cli.serviceToken
+	}
+
+	path := fmt.Sprintf("%s/%s/jobs", cli.hcCli.URL, cli.apiVersion)
```

Comment:
The options will each need to be added to the path if they are present. It needs to end up looking something like this: http://localhost:25700/jobs?offset=1&limit=7&sort=last_updated

Reply:
handled options for the endpoint now